plugins/org/src/components/Cards/Group/MembersList/MembersListCard.tsx

```diff
@@ -119,6 +119,15 @@ const MemberComponent = (props: { member: UserEntity }) => {
   );
 };
 
+const useListStyles = makeStyles(() => ({
+  root: {
+    height: '100%',
+  },
+  cardContent: {
+    overflow: 'auto',
```

Comment:
I see there's now always a scroll bar on https://backstage-default-pr-21304-c1940.uclusters.app.uffizzi.com/catalog/default/group/team-a where it doesn't overflow (or shouldn't, at least) the container - can that be addressed?

Reply:
because I don't specify the `MembersListCard` container height. You can do this in the local env

<img width="1112" alt="image" src="https://github.com/backstage/backstage/assets/7720722/9e6d7592-92d8-4c1a-93c0-77dfec96853e">